### PR TITLE
Added Google maps places api (Giving error)

### PR DESCRIPTION
**Pull request by Mohmedvaid**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,19 +44,21 @@
...
   </div>
 
   <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.15/lodash.core.min.js"></script>
   <script src="assets/js/module-herson.js"></script>
-  <script src="assets/js/module-marc.js"></script>
-  <script src="assets/js/module-garrett.js"></script>
+  <!-- <script src="assets/js/module-marc.js"></script>
+  <script src="assets/js/module-garrett.js"></script> -->
```
Comment (gmg300) [inline index.html]: Can you comment these back in?

**Review by gmg300 — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -44,19 +44,21 @@
...
-
     </div>
   </div>
 
   <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.15/lodash.core.min.js"></script>
   <script src="assets/js/module-herson.js"></script>
-  <script src="assets/js/module-marc.js"></script>
-  <script src="assets/js/module-garrett.js"></script>
```
Comment (Mohmedvaid) [inline index.html]: Commented out

File: index.html [hunk 3]
```diff
@@ -57,6 +54,11 @@
   <script src="assets/js/module-marc.js"></script>
   <script src="assets/js/module-garrett.js"></script>
   <script src="assets/js/module-mohmed.js"></script>
+    <!-- Compiled and minified CSS -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+
+    <!-- Compiled and minified JavaScript -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
```
Comment (gmg300) [inline index.html]: I don't think we need these since we have them above on line 51 and 52

**Review by gmg300 — CHANGES_REQUESTED** [review]

**Review by Mohmedvaid — COMMENTED** [review]
Fixed

File: index.html [hunk 4]
```diff
@@ -44,9 +44,6 @@
           lime wedge.</p>
       </div>
     </div> -->
-
-    <div id="g-maps">
-
     </div>
```
Comment (gmg300) [inline index.html:47]: Delete this as well

**Review by gmg300 — CHANGES_REQUESTED** [review]

**Review by Mohmedvaid — COMMENTED** [review]
added gmaps

**Review by gmg300 — APPROVED** [review]

**Review by Mohmedvaid — COMMENTED** [review]
added g-maps

**Pull request merged**